### ISSUE TITLE
Strikethrough not formatted

### ISSUE THREAD

**Issue by thvasilo** [SublimeText-Markdown/MarkdownEditing#529]
I've noticed this since a couple of months ago and I'm wondering if anyone else is experiencing it.

Text like 

```
~~This should happen~~
```

Does not get result in strike through formatting.

My user settings for GFM looks like this:

```
{
	"tab_size": 2,
	"draw_centered": false,
	"color_scheme": "Packages/MarkdownEditing/MarkdownEditor-Dark.tmTheme",
	"line_numbers": true,
	"mde.list_indent_bullets": ["*", "*", "*"],
}
```

**Comment by felixhao28** [comment]
This is how it looks on my machine.
![image](https://user-images.githubusercontent.com/2417391/42071953-b980c83e-7b90-11e8-92c1-550acaf13bb0.png)

What behavior do you expect?

**Comment by felixhao28** [comment]
The issue is fixed in e7430020e9d2eafd725618942de8bebd76f9aa6b. And it will roll out in the next release. If it really bothers you, you can [download and extract 2.2.8 branch](https://github.com/SublimeText-Markdown/MarkdownEditing/archive/2.2.8.zip) and follow [manual installation guide](https://github.com/SublimeText-Markdown/MarkdownEditing#manual-installation) from step 3.

**Comment by thvasilo** [comment]
Thanks, I'll close this for now and wait for the new release!
